packages/reports/addon/components/report-builder.hbs

```diff
@@ -43,9 +43,20 @@
             />
           </button>
           {{#unless @isFiltersCollapsed}}
-            <span class="report-builder__container-header__message flex align-items-center">
-              Click <DenaliIcon @icon="filter-add" @size="small" class="report-builder__container-header__message--icon m-x-2" /> next to a dimension/metric to add a filter.
-            </span>
+            <DenaliInputGroup @label="Row Limit" class="report-builder__row-limit">
+              {{#if (gt this.request.limit 10000)}}
+                <DenaliIcon @icon="warning" @size="small" id="report-builder__row-limit-warning" class="report-builder__row-limit-warning--icon"/>
+                <EmberTooltip targetId="report-builder__row-limit-warning" @popperContainer="body">
+                  You will likely have performance issues above 10,000 rows, use at your own risk.
+                </EmberTooltip>
+              {{/if}}
+              <DenaliInput placeholder="Row Limit"
+                @size="small"
+                @wrapperClass="w-20"
+                type="number"
```

Comment:
```suggestion
                type="number"
                min=1""
                max="10000"
```

Reply:
did we discuss allowing them to set it above 10k? And just showing a warning?